src/domain/session/room/timeline/MessageBodyBuilder.js

```diff
@@ -0,0 +1,96 @@
+import { linkify } from "./linkify/linkify.js";
+
+export class MessageBodyBuilder {
+
+    constructor(message = []) {
+        this._root = message;
+    }
+
+    fromText(text) {
+        const components = text.split("\n");
+        components.flatMap(e => ["\n", e]).slice(1).forEach(e => {
```

Comment:
flatMap is not supported by IE11, the legacy build _should_ include a polyfill for it from corejs and put it in the legacy bundle. We should double check this is actually happening before merging.